frontend/src/scenes/retention/RetentionTable.tsx

```diff
@@ -133,25 +129,24 @@ export function RetentionTable({
                                             </tr>
                                             <tr>
                                                 <td />
-                                                {results &&
-                                                    results[selectedRow]?.values.map((data: any, index: number) => (
+                                                {people.result?.totals &&
+                                                    people.result.totals.map((count: any, index: number) => (
                                                         <td key={index}>
-                                                            {data.count}&nbsp;{' '}
-                                                            {data.count > 0 && (
+                                                            {count}&nbsp;{' '}
+                                                            {count > 0 && (
                                                                 <span>
                                                                     (
-                                                                    {percentage(
-                                                                        data.count /
-                                                                            results[selectedRow]?.values[0]['count']
-                                                                    )}
+                                                                    {people.result
+                                                                        ? percentage(count / people.result.totals[0])
+                                                                        : percentage(0)}
                                                                     )
                                                                 </span>
                                                             )}
                                                         </td>
                                                     ))}
                                             </tr>
-                                            {people.result &&
-                                                people.result.map((personAppearances: RetentionTableAppearanceType) => (
+                                            {people.result?.detail &&
+                                                (people.result.detail as any[]).map((personAppearances) => (
```

Comment:
Is the `as` cast here needed? I think it isn't. At least it's removing type safety here.

Reply:
not sure what that was about. I think i was trying something and it got left in the commit